posthog/api/decide.py

```diff
@@ -93,11 +93,10 @@ def get_decide(request: HttpRequest):
         "isAuthenticated": False,
         # gzip and gzip-js are aliases for the same compression algorithm
         "supportedCompression": ["gzip", "gzip-js"],
+        "featureFlags": [],
+        "sessionRecording": False,
     }
 
-    response["featureFlags"] = []
-    response["sessionRecording"] = False
```

Comment:
fly-by I don't _think_ there's a reason not to declare then in with the dict creation